flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/source/InfluxDBSourceBuilder.java

```diff
@@ -124,14 +109,20 @@
      * @return a InfluxDBSource with the settings made for this builder.
      */
     public InfluxDBSource<OUT> build() {
-        this.sanityCheck();
         return new InfluxDBSource<>(this.properties, this.deserializationSchema);
     }
 
     // ------------- private helpers  --------------
-    private void sanityCheck() {
```

Comment:
Why did you delete the sanityCheck here?

Reply:
Ups! I was testing something and removed it. I will add it back.